src/game/g_local.h

```diff
@@ -658,6 +658,7 @@ typedef struct
 	int spec_invite;
 	int spec_team;
 	int kills;
+	int kills_assists;
```

Comment:
`kills_assists` to me implies this included both kills and assists, I'd just use either `kill_assists` or `assists` for this.
```suggestion
	int kill_assists;
```
```suggestion
	int assists;
```